Description: Adds /usr/share/openstack-dashboard in the search path for the django.wsgi
Author: Mehdi Abaakouk <sileht@sileht.net>
Forwarded: not-needed

Index: horizon/openstack_dashboard/wsgi/django.wsgi
===================================================================
--- horizon.orig/openstack_dashboard/wsgi/django.wsgi
+++ horizon/openstack_dashboard/wsgi/django.wsgi
@@ -9,6 +9,8 @@ sys.path.insert(0, os.path.join(os.path.
 os.environ['DJANGO_SETTINGS_MODULE'] = 'openstack_dashboard.settings'
 sys.stdout = sys.stderr
 
+sys.path.append("/usr/share/openstack-dashboard/")
+
 DEBUG = False
 
 application = get_wsgi_application()
